app/Http/Requests/StoreNews.php

```diff
@@ -3,30 +3,36 @@
 namespace App\Http\Requests;
 
 use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rule;
 
 class StoreNews extends FormRequest
 {
-  /**
-   * Determine if the user is authorized to make this request.
-   *
-   * @return bool
-   */
-  public function authorize()
-  {
-    return true;
-  }
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
```

Comment:
whitespace changes. Let's keep the formatting consistent (psr-2 or laravel recommendations) and revert unnecessary whitespace changes